### DISCUSSION TITLE
能管理已有的证书么？

### DISCUSSION THREAD

**Discussion by yusiwen** [0xJacky/nginx-ui#726] [General]
我自己的VPS，已经有若干virtual site部署着，都是用certbot手工获取的let's encrypt证书，并定时更新的。今天安装了Nginx-UI，成功后登录看到证书列表里是空的，站点列表是正确的。

请问：Nginx-UI能否管理Host上已经存在的Let'sEncrypt证书？

**Comment by 0xJacky** [comment]
您好，不可以的，只能托管由 Nginx UI 申请单证书。
